clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -1407,6 +1407,36 @@ def GlobalAnnotationValuesAttr : CIR_Attr<"GlobalAnnotationValues",
   let genVerifyDecl = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// AddressPointAttr
+//===----------------------------------------------------------------------===//
+
+def AddressPointAttr : CIR_Attr<"AddressPoint", "address_point"> {
+  let summary = "Address point attribute";
+
+  let description = [{  
+  Attribute specifying the address point within a C++ virtual table (vtable).
+
+  The `vtable_index` parameter identifies which vtable to use within a vtable
+  group, while the `address_point_index` specifies the offset within
+  that vtable where the address begins.
```

Comment:
Because the attribute name contains "address point", let's rename the operands a bit, how about:
- vtable_index -> `index`
- address_point_index -> `offset`
You can keep the original names in the desc, example: "The \`index`\ (vtable index) parameter identifies ...", same for the other one.